scripts/multiple_databus_registration.py

```diff
@@ -1,28 +1,23 @@
 import os
 import pathlib
-
-import environ_setup
+from dotenv import load_dotenv
 import requests
 
-environ_setup.setup()
-
-MODEX_URL = "https://modex.rl-institut.de/databus/"
+load_dotenv()
 
-path = pathlib.Path(
-    "/home/local/RL-INSTITUT/felix.maurer/rli/Felix.Maurer/SEDOS/Python/data_adapter_oemof/"
-    "tests/_files/tabular_datapackage_mininmal_example_collection/csvs"
-)
+path = pathlib.Path("Path to CSVs")
 
 CSVS = list(path.iterdir())
 
-account = os.environ["account_name"]
-api_key = os.environ["API_KEY"]
-group = os.environ["group"]
-version = os.environ["version"]
+account = os.getenv("account")
+api_key = os.getenv("api_key")
+group = os.getenv("group")
+version = os.getenv("version")
```

Comment:
See above - those could be parameters to a function (with default `None`) - if `None` is set os.environ is checked